doc/nrf/ug_zigbee_other_ecosystems.rst

```diff
@@ -67,4 +67,4 @@ The following instructions can help you troubleshoot the connection problem:
 #. Make sure that the device is scanning through all channels to find the Zigbee network.
    For example, verify that the :kconfig:`CONFIG_ZIGBEE_CHANNEL_SELECTION_MODE_MULTI` Kconfig option is correctly set.
 #. Check if the device is compatible with Zigbee 3.0.
-   If it *not* compatible, enable the legacy mode by including a call to :c:func:`zb_bdb_set_legacy_device_support`.
+   If it is *not* compatible, enable the legacy mode by including a call to :c:func:`zb_bdb_set_legacy_device_support`.
```

Comment:
Should the `zb_bdb_set_legacy_device_support` function link to [this page](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/3.9.0.1/group__zboss__bdb__comm__params.html#ga8bf3b3beef192c00bcdca17ad1240921)?

Reply:
It should. Can you add the link? Thanks for spotting this.

Reply:
I would say yes.